matter/efr32/efr32mg12/BRD4161A/autogen/sl_event_handler.c

```diff
@@ -20,6 +20,7 @@
 #if defined(CONFIG_ENABLE_UART)
 #include "sl_uartdrv_instances.h"
 #endif // CONFIG_ENABLE_UART
+#include "sl_spidrv_instances.h"
```

Comment:
This include should be wrap with a #ifdef SL_WIFI.

 same for `sl_spidrv_init_instances();` and this applies to all sl_event_hanlder.c of the boards support you are adding it too.
 
 BRD4161A
 BRD4162A
 BRD4186C
 BRD4187C

Reply:
done